modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -36,70 +46,131 @@ export default class ortcRTCPeerConnection extends yaeti.EventTarget {
 
-        // Create a RTCIceTransport.
+        // RTCIceTransport.
         // @type {RTCIceTransport}
-        this._iceTransport = this._createIceTransport(this._iceGatherer);
+        this._iceTransport = null;
+
+        // RTCDtlsTransport.
+        // @type {RTCDtlsTransport}
+        this._dtlsTransport = null;
+
+        // Local RTP capabilities (filtered with remote ones).
+        // @type {RTCRtpCapabilities}
+        this._localCapabilities = null;
 
         // Local RTCSessionDescription.
         // @type {RTCSessionDescription}
         this._localDescription = null;
 
-        // Set of local MediaStreams.
-        // @type {Set<MediaStream>}
-        this._localStreams = new Set();
+        // Map with info regarding local media.
+        // - index: MediaStreamTrack.id
+        // - value: Object
+        //   - rtpSender: Associated RTCRtpSender instance
+        //   - stream: Associated MediaStream instance
+        //   - ssrc: Provisional or definitive SSRC
+        //   - rtxSsrc: Provisional or definitive SSRC for RTX
+        //   - sending: Boolean indicating whether rtpSender.send() was called.
+        this._localTrackInfos = new Map();
+
+        // Ordered Map with MID as key and kind as value.
+        // @type {map<String, String>}
+        this._mids = new Map();
 
         // Remote RTCSessionDescription.
         // @type {RTCSessionDescription}
         this._remoteDescription = null;
 
-        // Set of remote MediaStreams.
-        // @type {Set<MediaStream>}
-        this._remoteStreams = new Set();
+        // Map of remote streams.
+        // - index: MediaStream.jitsiRemoteId (as signaled in remote SDP)
+        // - value: MediaStream (locally generated so id does not match)
+        // @type {map<Number, MediaStream>}
+        this._remoteStreams = new Map();
+
+        // Map with info about receiving media.
+        // - index: Media SSRC
+        // - value: Object
+        //   - kind: 'audio' / 'video'
+        //   - ssrc: Media SSRC
+        //   - rtxSsrc: RTX SSRC (may be unset)
+        //   - streamId: MediaStream.jitsiRemoteId
+        //   - trackId: MediaStreamTrack.jitsiRemoteId
+        //   - cname: CNAME
+        //   - stream: MediaStream
+        //   - rtpReceiver: Associated RTCRtpReceiver instance
+        // @type {map<Number, Object>}
+        this._remoteTrackInfos = new Map();
+
+        // Local SDP global fields.
+        this._sdpGlobalFields = {
+            id: RandomUtil.randomInt(1000000, 9999999),
+            version: 0
+        };
 
         // RTCPeerConnection signalingState.
         // @type {RTCSignalingState}
         this._signalingState = RTCSignalingState.stable;
+
+        // Create the RTCIceGatherer.
+        this._setIceGatherer(pcConfig);
+
+        // Create the RTCIceTransport.
+        this._setIceTransport(this._iceGatherer);
+
+        // Create the RTCDtlsTransport.
+        this._setDtlsTransport(this._iceTransport);
     }
```

Comment:
I don't see any point here. Nobody making WebRTC stuff since 2015 expects non rtcp-mux. Chrome and Firefox are about removing support for it, and rtcp-mux is marked as "implementation risk" in the WG.